src/kemal/param_parser.cr

```diff
@@ -60,6 +66,17 @@ class Kemal::ParamParser
     end
   end
 
+  def parse_msgpack
+    body = @request.body as String
+    unpacker = MessagePack::Unpacker.new(body.bytes)
+    case data = unpacker.read
+    when Hash
+      data.each do |k, v|
+        @params[k as String] = v as AllParamTypes
```

Comment:
Also since `AllParamTypes` do not include the unsigned type it would not work for everything right ?